recipes-support/snapd/snapd_2.47.1.bb

```diff
@@ -44,7 +44,6 @@ S = "${WORKDIR}/${PN}-${PV}"
 
 EXTRA_OECONF += "			\
 	--disable-apparmor		\
-	--enable-seccomp		\
```

Comment:
I see that libseccomp was dropped and seccomp-support in AppArmor was disabled. However, we actually want at least seccomp to work, so I'd rather this change was reverted.

At the same time meta-security has libseccomp updated to 2.5 without the bugs with 64bit argument handling. Can you pull in the update and replace the 2.3 lib in meta-snappy? See: https://git.yoctoproject.org/cgit/cgit.cgi/meta-security/tree/recipes-security/libseccomp